test/test_sparse.py

```diff
@@ -1233,14 +1238,17 @@ def test_shape(m, n, p, nnz, broadcast):
             S = self._gen_sparse(2, nnz, [n, m], dtype, device, coalesced)[0]
             S_dense = S.to_dense().requires_grad_(True)
             S.requires_grad_(True)
-            self.assertEqual(torch.sparse.addmm(D1, S, D2), torch.addmm(D1, S_dense, D2))
+            Y = torch.sparse.addmm(D1, S, D2, beta=beta, alpha=alpha)
+            Y_dense = torch.addmm(D1, S_dense, D2, beta=beta, alpha=alpha)
+            self.assertEqual(Y, Y_dense)
 
-            def fn(S, D1, D2):
-                return torch.sparse.addmm(D1, S, D2)
+            def fn(S, D1, D2, beta=beta, alpha=alpha):
+                return torch.sparse.addmm(D1, S, D2, beta=beta, alpha=alpha)
             gradcheck(fn, (S, D1, D2), check_sparse_nnz=True)
 
-        test_shape(7, 8, 9, 20, False)
-        test_shape(7, 8, 9, 20, True)
+        test_shape(7, 8, 9, 20, False, None)
+        test_shape(7, 8, 9, 20, True, None)
+        test_shape(7, 8, 9, 20, False, (1, 0))
```

Comment:
Can you please add test with `True, (1,0)` and True and False (1,1) to explicitly test most common alpha/beta combinations?